clients/clojure/LICENSE

```diff
@@ -0,0 +1,280 @@
+Eclipse Public License - v 2.0
```

Comment:
@harikrishnakanchi License should be Apache 2.0

Reply:
updated to apache 2.0